application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -45,7 +45,7 @@
     private static final String MAX_USES_OPTION = "max-uses";
     private static final String MAX_AGE_OPTION = "max-age";
 
-    private static final String YOUTUBE_TOGETHER_NAME = " YouTube Together";
+    private static final String YOUTUBE_TOGETHER_NAME = "YouTube Together";
```

Comment:
Make this constant public like the others.

Reply:
@illuminator3 ok I will do that 😌

Reply:
After changing I have to do pull request again?? Or any other thing is there?

Reply:
After changing it, you simply have to commit your change and then push it. The PR will automatically be updated.